### PR TITLE
Update draft_saml.asciidoc

### PR DESCRIPTION
**Pull request by LMS235**
- Mellon Module from OMD (since 2.0)
- Adjust Rewrite Filter

### CODE REVIEW

**Comment by bili123** [comment]
Thanks for the contribution!

We had a more up to date version of this article in the master branch (without the draft_ prefix) that was not yet migrated to the 2.0.0 branch when you PRed - because it was still in review and translation. As of now, everything is up to date in both branches and languages (the user guide website will be updated tonight).  The adjustments of the rewrite filter and the note to the Mellon module were already implemented through earlier PRs.


Long story short: I'll close this PR without merging, but your content is in!
